src/pages/Recruiters.vue

```diff
@@ -0,0 +1,595 @@
+<script>
+import { get, values, pick } from 'lodash'
+import { mapState, mapGetters } from 'vuex'
+import { date } from 'quasar'
+
+import EventBus from 'src/utils/event-bus'
+import { extractLocationDataFromPlace } from 'src/utils/places'
+import logger from 'src/utils/logger'
+
+import HeroTwoColLayout from 'src/layouts/HeroTwoColLayout'
+
+import AppGalleryUploader from 'src/components/AppGalleryUploader'
+import CustomAttributesEditor from 'src/components/CustomAttributesEditor'
+import DateRangePicker from 'src/components/DateRangePicker'
+import PlacesAutocomplete from 'src/components/PlacesAutocomplete'
+import SelectAssetType from 'src/components/SelectAssetType'
+import SelectCategories from 'src/components/SelectCategories'
+
+import PageComponentMixin from 'src/mixins/pageComponent'
+
+export default {
```

Comment:
We won’t be able to maintain duplicates of pages.

I suggest you use the same `NewAsset.vue` page you can also restyle if needed (depending on `this.$route`)

Reply:
No problem. Thank you